app/models/form_profile.rb

```diff
@@ -189,15 +189,19 @@ def self.load_form_mapping(form_id)
   # * TODO(AJD): MIS (military history)
   #
   def prefill
-    @identity_information = initialize_identity_information
-    @contact_information = initialize_contact_information
-    @military_information = initialize_military_information
-    mappings = self.class.mappings_for_form(form_id)
-
     form = form_id == '1010EZ' ? '1010ez' : form_id
-    form_data = generate_prefill(mappings) if FormProfile.prefill_enabled_forms.include?(form)
```

Comment:
hmm 🤔 wouldn't `form_data` be `nil` here if your form id wasn't included in `prefill_enabled_forms`?

Reply:
@rmtolmach Thanks for taking a look! I'd imagine it would be `nil`. It was failing before this line, previously. It failed on line 195 with the call to `mappings_for_form`. That's why I wanted to wrap in the in conditional on whether the form is prefill enabled. Does that make sense?

Reply:
oh, I see. For some reason it was making me nervous that you were putting `@identity_information`, etc. inside the conditional, but I think you'd only need to initialize those if prefill was needed for the form.

I just discovered another way to get around this problem: https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/form_profile_mappings/20-0996.yml#L10

Reply:
Oh yes that makes sense. I'll move those instance variables outside of the conditional to reduce risk.

I'm not sure if the `nonPrefill` approach is best for us because that would encompass the entire form, for all of our forms. We'd end up inserting a lot of code/config for very little functionality, I think. Do you prefer that I do that rather than my approach here?